scripts/hca-utils/hca_utils/utils.py

```diff
@@ -0,0 +1,332 @@
+import csv
+from functools import lru_cache
+import os
+from requests.exceptions import HTTPError
+from typing import Set
+import urllib.parse
+
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from google.cloud import bigquery, storage
+from requests_cache.core import CachedSession
+
+
+class HcaUtils:
+    def __init__(self, environment: str, project: str, dataset: str):
+        self.environment = environment
+
+        if environment == "dev":
+            self.project = "broad-jade-dev-data"
```

Comment:
if we force-overwrite the value of `project` if the env is dev, i think that `project` should be an optional param in the initializer here and be rejected by the argument parser if it's passed in when the env is dev. that'll better surface what's happening to users of the script